### PR TITLE
Examplefailsafe

### PR DESCRIPTION
**Pull request by DrCherry**
This closes issue #295 
If the default username is left, the login method exits with a nice error explaining what went wrong.

### CODE REVIEW

**Comment by Killerherts** [comment]
after i figure out how to run a posix ill test it

**Comment by DrCherry** [comment]
http://www.howtogeek.com/249966/how-to-install-and-use-the-linux-bash-shell-on-windows-10/

**Comment by Killerherts** [comment]
windows 7 does not have a subsystem can this be done with cygwin

**Comment by DrCherry** [comment]
It seems like cygwin would work but I cannot confirm.

**Comment by Killerherts** [comment]
colors do work in cygwin its pain to start the script i had no issues with your fork tonight

**Comment by polaroi8d** [comment]
@DrCherry can you rebase all your commit to one?

File: example.py [hunk 1]
```diff
@@ -17,6 +17,7 @@
                tag_list=['follow4follow', 'f4f', 'cute'],
                tag_blacklist=['rain', 'thunderstorm'],
                user_blacklist={},
+               user_whitelist=['nik.kle'],
```
Comment (polaroi8d) [inline example.py]: IMHO this is not necessary for the general code.

**Comment by DrCherry** [comment]
It's been a while since I've rebased.  I'll give it a shot.

**Comment by LevPasha** [comment]
Guys, test it and i will merge if all ok. THX!

File: example.py (hunk 1 above)
Comment (Killerherts) [inline example.py]: This Feature has been asked for alot lately

**Comment by DrCherry** [comment]
I've created a new fork and pull request because I was wasting too much time trying to rebase.  It should be tested a bit more and I'll be happy to patch up any bugs I've created.

**Pull request closed without merging**